rust/libnewsboat/src/fmtstrformatter/limited_string.rs

```diff
@@ -32,16 +22,16 @@ impl LimitedString {
         }
     }
 
-    /// Returns the number of graphemes in the string.
+    /// Returns the "displayed width" of the string
     pub fn length(&self) -> usize {
-        utils::graphemes_count(&self.content)
```

Comment:
Are there any other users of `utils::graphemes_count`? Can that function be removed now?